src/main/kotlin/com/github/igorperikov/mightywatcher/service/OutputService.kt

```diff
@@ -0,0 +1,85 @@
+package com.github.igorperikov.mightywatcher.service
+
+import com.github.igorperikov.mightywatcher.Issues
+import com.github.igorperikov.mightywatcher.entity.NamedTimestamp
+import j2html.TagCreator.*
+import j2html.attributes.Attr
+import j2html.tags.ContainerTag
+import org.slf4j.LoggerFactory
+import java.io.File
+import java.time.LocalDate
+import java.time.format.DateTimeFormatter
+import java.util.*
+
+const val PDF_OUTPUT_TYPE = "PDF"
+const val HTML_OUTPUT_TYPE = "HTML"
```

Comment:
I'd like to have it implemented as a different descendants of `OutputService` interface, rather "everything in the one class with a switch expression", easier to test as well